MobiusCore/Test/MobiusControllerTests.swift

```diff
@@ -233,24 +218,20 @@ class MobiusControllerTests: QuickSpec {
                         controller.connectView(view)
                         controller.start()
 
-                        controller.disconnectView()
-                        expect(errorThrown).to(beTrue())
+                        expect(controller.disconnectView()).to(raiseError())
                     }
                 }
 
                 #if arch(x86_64) || arch(arm64)
```

Comment:
I don’t know why this is here, it predates the public repo. Possibly it was from a previous attempt to use Nimble’s `throwAssertion()` predicate.

Reply:
Yes, I remember there was a need to use this kind of #if statements to only run tests when Nimble supported it.